lib/src/qr_image.dart

```diff
@@ -233,10 +233,10 @@ class _QrImageState extends State<QrImage> {
   }
 
   Widget _errorWidget(
-      BuildContext context, BoxConstraints constraints, Object error) {
+      BuildContext context, BoxConstraints constraints, Object? error) {
     final errorWidget = widget.errorStateBuilder == null
         ? Container()
-        : widget.errorStateBuilder(context, error) ?? Container();
+        : widget.errorStateBuilder!(context, error) ?? Container();
```

Comment:
I used this branch in my app to test it a bit. My app seemed to work fine but duing assembly flutter was yelling at me:
```
../../snap/flutter/common/flutter/.pub-cache/git/qr.flutter-2648de0844ec9a2b9e463edc527d29362506105d/lib/src/qr_image.dart:239:36: Warning: Operand of null-aware operation '??' has type 'Widget' which excludes null.
 - 'Widget' is from 'package:flutter/src/widgets/framework.dart' ('../../snap/flutter/common/flutter/packages/flutter/lib/src/widgets/framework.dart').
        : widget.errorStateBuilder!(context, error) ?? Container();
```

The return type of   QrErrorBuilder is Widget so the ?? Container() is not necessary here.

Reply:
fixed